crates/rolldown_utils/src/filter_expression.rs

```diff
@@ -90,16 +91,53 @@ pub enum Token {
   Include,
   Exclude,
   CleanUrl,
+  String(String),
```

Comment:
Extract atom token as a standalone token, aiming to make the `Token` type easier to extend.

Reply:
preparation for the `Query` filter rust impl